src/test/java/services/WaitService.java

```diff
@@ -0,0 +1,50 @@
+package services;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import utils.PropertyReader;
+
+import java.util.List;
+
+public class WaitService {
+
+    private WebDriverWait wait;
+
+    public WaitService(WebDriver driver) {
+        this.wait = new WebDriverWait(driver, PropertyReader.getDefaultWebDriverWait());
+    }
+
+    public WebElement getClickableElement(WebElement webElement)
+    {
+        try {
+            return wait.until(ExpectedConditions.elementToBeClickable(webElement));
+        }
+        catch (Exception e) {
```

Comment:
Are you sure this is the best way to handle exception? what kind of exception can you get at this place? is it always NoSuchElement or might be a different type? 
With current implementation no matter what kind of exception you are getting, you throw NoSuchElement... this doesn't give you relevant details about the error...

Reply:
I'll fix it in classes for wrapping elements later.